pkgs/applications/misc/gkrellm/default.nix

```diff
@@ -1,8 +1,21 @@
 { lib, fetchurl, stdenv, gettext, pkg-config, glib, gtk2, libX11, libSM, libICE, which
+, makeDesktopItem, wrapGAppsHook
 , IOKit ? null }:
```

Comment:
```suggestion
, IOKit }:
```

Reply:
IOKit is not defined for me.

Reply:
It can't be because https://github.com/NixOS/nixpkgs/blob/21a675946347bca090e2e68bfb5b1677b6113cb2/pkgs/top-level/all-packages.nix#L22490 is wrong. It should be ``inherit (darwin.apple_sdk.frameworks) IOKit;``

Reply:
Done.